101-automation-with-modules-and-runbook-execution/azuredeploy.json

```diff
@@ -34,16 +34,16 @@
     "Password": {
       "type": "securestring",
       "metadata": {
-        "description": "aduser password"
+        "description": "Azure AD Password to authenticate into automation runbook"
       }
     },
-	 "WebAppName": {
+    "WebAppName": {
```

Comment:
you'll need to add prereq templates (see contribution guide) to get a passing CI